wolfcrypt/src/asn.c

```diff
@@ -7962,19 +7968,8 @@ int PemToDer(const unsigned char* buff, long longSz, int type,
 
     headerEnd += XSTRLEN(header);
 
-    if ((headerEnd + 1) >= bufferEnd)
-        return BUFFER_E;
-
-    /* eat end of line */
-    if (headerEnd[0] == '\n')
-        headerEnd++;
-    else if (headerEnd[1] == '\n')
-        headerEnd += 2;
-    else {
-        if (info)
-            info->consumed = (long)(headerEnd+2 - (char*)buff);
```

Comment:
info->consumed marker was added for a reason, why are we removing it?

Reply:
Its not removed, it get updated correctly in the lines below.